### DISCUSSION TITLE
Can another organization use the Equinor docker images?

### DISCUSSION THREAD

**Discussion by akosjoebodor** [equinor/witsml-explorer#294] [Q&A] [answered]
Hi,
I tried to run the Equinor docker images based on the guide:
https://github.com/equinor/witsml-explorer/blob/main/Docker/run-with-docker.md

I created the Personal Access Token and could login with docker but when docker-compose tries to pull the images I got the error below
(I don't have SSO button in Personal Access Token section in Github):
```
Pulling api (docker.pkg.github.com/equinor/witsml-explorer/witsmlexplorer-api:latest)...
ERROR: repository docker.pkg.github.com/equinor/witsml-explorer/witsmlexplorer-api not found: name unknown: docker package "witsmlexplorer-api" does not exist under owner "equinor/witsml-explorer"
```

Is this because the witsmlexplorer-api and witsmlexplorer-frontend docker images are private images and only used by Equinor?
If we want to dockerize the WE should we create our own docker images?

Thanks for your help!

**Comment by akosjoebodor** [comment, marked as answer]
Meanwhile I created our own docker images with the script **build_docker_images.sh** in the repo.

**Reply by lindsve** [reply]
This is the correct answer. The pre-built Equinor images are for internal use only, but it sounds like you got it working
